app/src/main/res/layout/activity_main.xml

```diff
@@ -31,7 +31,7 @@
         android:layout_marginStart="115dp"
         android:layout_marginEnd="115dp"
         android:layout_marginBottom="227dp"
-        android:backgroundTint="#2196F3"
+        android:backgroundTint="#8B9CBC"
```

Comment:
i think we should probably use the colors.xml for this kind of stuff. if the color doesn't exist there yet, you can add it

Reply:
same for the other stuff on the page

Reply:
l8r problem merge rn

Reply:
gimme green checkmark @stephchi0

Reply:
kk